android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.reactnativeshopify">
```

Comment:
This probably needs to stay in sync with the option from the `BuyClientBuilder`. Have you checked if this needs to be configurable or not?